tests/test_spatial_resample.py

```diff
@@ -142,7 +142,7 @@ def test_flips(self, img, device, data_param, expected_output):
                 img = img.to(device)
             out = SpatialResample()(img=img, **data_param)
             assert_allclose(out, expected_output, rtol=1e-2, atol=1e-2)
-            assert_allclose(out.affine, data_param["dst_affine"])
+            assert_allclose(to_affine_nd(len(out.shape) - 1, out.affine), data_param["dst_affine"])
 
     @parameterized.expand(TEST_4_5_D)
     def test_4d_5d(self, new_shape, tile, device, dtype, expected_data):
```

Comment:
Hi @wyli , here I'm a bit confused with this test case. 
According to: https://github.com/Project-MONAI/MONAI/pull/6031/files#diff-3a1b9fb96f9cde878a33605fcbd39cf1af51a79c285fe662d80a00f0c43e2d5fL205
the transform does not support >3d input images, is it correct?